scripts/west_commands/blobs.py

```diff
@@ -112,8 +114,9 @@ def get_blobs(self, args):
 
     def list(self, args):
         blobs = self.get_blobs(args)
+        fmt = getattr(args, 'format', self.DEFAULT_LIST_FMT)
```

Comment:
Why not use the `default=` parameter to `add_argument()` instead?

Reply:
> Why not use the `default=` parameter to `add_argument()` instead?

Because that would break this line: https://github.com/zephyrproject-rtos/zephyr/pull/49253/files#diff-00f84402431f5f9e22825d030bf0149f3c749cb524804e9b81ca81049088607fR158